addon/services/gwr.js

```diff
@@ -113,7 +113,8 @@ export default class GwrService extends Service {
     })[searchKey];
   }
 
-  extractErrorsFromXML(xml, specificError, specificErrorMsg) {
+  // (xml: String, errorsToMatch: [[errorKey: String,errorMsg: String]]) => [String]
```

Comment:
Shouldn't the inputs be an array of arrays in this case?

Reply:
Done